client/extensions/woocommerce/woocommerce-services/components/text-field/index.js

```diff
@@ -24,8 +24,9 @@ const TextField = ( {
 	updateValue,
 	error,
 	className,
+	defaultValue
```

Comment:
Same name of the React's default prop, for consistency: https://reactjs.org/docs/uncontrolled-components.html#default-values